pipelines.md

```diff
@@ -81,17 +83,73 @@ The default tasks and pipelines can be updated by forking the Kabanero Pipelines
+- [validate-active-stack-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/validate-active-stack-task.yaml)
+
+This task will validate the stack is allowed to build an deploy on the cluster.  It checks the digest of the stack image specified in the `.appsody-config.yaml` of the project and validates it matches the digest of the stack image that's active on the collection.  If they do not match, the pipeline will fail and none of the other steps will execute.
 
-   This file builds a container image from the artifacts in the git-source repository by using [Buildah](https://github.com/containers/buildah). After the image is built, the `build-task` file publishes it to the docker-image URL by using Buildah.
+- [build-push-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/build-push-task.yaml)
+
+   This file builds a container image from the artifacts in the git-source repository by using `appsody build`.  The appsody build command will leverage the [Buildah](https://github.com/containers/buildah) options. After the image is built, the image will get published to the container registry that is configured. The build-push-task will also generate the `app-deploy.yaml` that is used by the `deploy-task`.  If there is already a copy of the `app-deploy.yaml` file in the source repository, it will get merged with the new one genrated by this step.
 
 - [deploy-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/deploy-task.yaml)
 
-   This file modifies the `app-deploy.yaml` file, which describes the deployment options for the application. `Deploy-task` modifies `app-deploy.yaml` to point to the image that was published and deploys the application by using the application deployment operator. Generate your `app-deploy.yaml` file by running the `appsody deploy --generate-only` command.
+   `Deploy-task` uses `app-deploy.yaml` to deploy the application to the cluster by using the application deployment operator. By default, the pipelines run and deploy the application in the `kabanero` namespace. If you want to deploy the application in a different namespace, update the `app-deploy.yaml` file to point to that namespace.
 
-By default, the pipelines run and deploy the application in the `kabanero` namespace. If you want to deploy the application in a different namespace, update the `app-deploy.yaml` file to point to that namespace.
+- [image-scan-task.yaml](https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/incubator/image-scan-task.yaml)
 
-For more information, see [the kabanero-pipelines repo](https://github.com/kabanero-io/kabanero-pipelines).
+  The `image-scan-task` will initiate a container scan of the image published by the `build-push-task` using OpenSCAP.  The results of the scan get published in the logs of the task.
```

Comment:
We need to indicate what `image-scan-task` is.  Is it a step? an option? a field?  Whatever it is, indicate after the name, like so:
Change
The `image-scan-task` will initiate
to
The `image-scan-task` option initiates

Change
The results of the scan get published
to
The results of the scan are published